analytics_nudge.md

```diff
@@ -0,0 +1,57 @@
+## Context and Goals
+
+Currently, we nudge people to opt into (or out of) analytics the first time they run Tilt. This is a bad time to nudge them -- they have no reason to trust us yet, and have seen no value from Tilt that might make them willing to share their data. Moreover, to opt in or out, the user has to run a separate CLI tool, `tilt analytics`, which is an awkward extra step and takes them out of flow. 
```

Comment:
I think there's a difference between a "nudge" and a "hurdle", and right now we hurdle. It's not important to the doc, but I think a distinction that can be worth making.

Reply:
good call, I tightened up this paragraph.